Makefile

```diff
@@ -18,6 +18,7 @@ clean:
 
 sync:
 	pipenv sync --dev
+	pipenv run pip install -e .
```

Comment:
Pipenv does not install tentaclio from the Pipefile 🙄

Reply:
:roll_eyes: